README.md

```diff
@@ -1,10 +1,51 @@
 # Drupal 8 Media Guide
 
-## How to contribute to the documentation
+##Introduction
+
+This guide provides and overview of the approach to Media in Drupal 8.
+
+The Drupal Media Team has set an ambitious [goal](roadmap.md) of ensuring Drupal offers best media solution available in any web framework. 
+
+We still have a ways to go to achieve that goal, but this guide will help you to understand our [Roadmap](roadmap.md) and [Module Status](modules/intro.md). 
+
+
+As discussed in the [Roadmap](roadmap.md) and throughout this Guide, the Media Team has worked collaboratively to separate Media functionality into several independent modules that can be used together based as appropriate. Overtime we'll be adding examples of how these modules can be integrated.
+
+We also hope that this guide will help you understand best practices and get started using media in your Drupal 8 project today.
+
+## Getting Involved
+
+The Drupal Media team is actively seeking contributions and new team members. There are many ways to get involved, including helping with funding, project management communications, documentation, testing and development. Please check out our [Contributing](contributing.md) page for full details.
+
+
+##Using this Guide
+
+As mentioned above, Media for Drupal 8 is undergoing a re-architecture into individual components. For example, the Media Browser functionality is being developed into the Entity Browser module and embedding media in WYSIWYG is being developed into the Entity Embed module.
+
+While not all of these components are complete you can still to a lot with what's done. This Guide includes a [Module Status](status.md) section which provide an overview of the ecosystem, and then more detailed guides for each of the available modules:
+
+* [Crop API](modules/crop/intro.md)
+* [DropzoneJS](modules/dropzonejs/intro.md)
+* [Embed API](modules/embed/intro.md)
+* [Entity Embed](modules/entity_embed/intro.md)
+* [Entity Browser](modules/entity_browser/intro.md)
+* [File Entity](modules/file_entity/intro.md)
+* [Media Entity](modules/media_entity/intro.md)
+* [URL Embed](modules/url_embed/intro.md).
+
+
+
+### Troubleshooting
 
 If you have trouble understanding any part of the documentation, please notify those of us who work on this section by [creating an issue](https://github.com/drupal-media/handbook/issues) and clearly explain what you don't understand and why - we're happy to hear from you, your contribution helps everyone at Drupal!
```

Comment:
This is deprecated link. Should we update to https://github.com/drupal-media/d8-guide/issues?